Install-Prerequisites.ps1

```diff
@@ -1,5 +1,5 @@
 # The psake module is needed to run tests and publish the module to powershell gallery.
 Set-PSRepository -Name "PSGallery" -InstallationPolicy Trusted
```

Comment:
This is pretty much NO-OP when using `-Scope CurrentUser -Force` at least on Linux

Reply:
Old comment on a older commit but yes